expyfun/_git.py

```diff
@@ -105,6 +113,11 @@ def assert_version(version):
         Version to check (7 characters).
     """
     _check_version_format(version)
+    if this_fork is not None and this_fork.lower() != 'labsn':
+        warnings.warn('You are running a forked version of the expyfun '
+                      'codebase (fork "{}"). Do not do this if you are '
+                      'running an actual experiment; only do this during '
+                      'development.'.format(this_fork), RuntimeWarning)
```

Comment:
I'm not sure how to test this, because the tests run within the system's expyfun directory.  I can download expyfun to a `_TempDir()` and I can `sys.path.insert` that dir at position zero, but the cwd will always get checked first before the rest of the path options, so I can't import anything from the forked-and-downloaded version of the codebase, hence I cannot test (within the unittests) whether this warning actually comes up.  Am I missing something, or do we have to live with this decline in coverage?